cmd/factory_test.go

```diff
@@ -127,8 +127,8 @@ func (t TestFactory) ConnectionNode() (*p2p.QriNode, error) {
 	return t.node, nil
 }
 
-// RPC returns from internal state
-func (t TestFactory) RPC() *rpc.Client {
+// HTTPClient returns from internal state
```

Comment:
This old comment seems like it was stale, or something. How about `HTTPClient returns nil for tests`